### ISSUE TITLE
Multiplier in instance specific optimization

### ISSUE THREAD

**Issue by xiaogengen-007** [voxelmorph/voxelmorph#463]
Hello,

**Task** (what are you trying to do/register?)
I am running https://github.com/voxelmorph/voxelmorph/blob/dev/scripts/tf/train_instance.py to do instance-specific optimization. It works pretty well. I am just wondering what the purpose of the "multiplier" is and how to set its proper value:
`parser.add_argument('--multiplier', type=float, default=1000, help='local weight multiplier (default: 1000)')`

**What have you tried**
I thought the "multiplier" (or the "biasmult" in ne.tf.layers.LocalParamWithInput) was just a scaling factor that is related to the learning rate. I tried (1) multiplier=1, lr=1; and (2) multiplier=1000, lr=1e-3. They both give reasonable registration results, but the latter one achieves much lower loss. It seems the multiplier affects more than the learning rate. Could you please give some insight on that?

Thank you so much in advance for any help!
